lib/gettext_i18n_rails_js/task.rb

```diff
@@ -0,0 +1,56 @@
+require "gettext_i18n_rails/tasks"
+
+module GettextI18nRailsJs
+  module Task
+    extend self
+
+    def files_list
+      Pathname.glob(
+        ::File.join(
+          locale_path,
```

Comment:
@tboerger No, I have required `gettext_i18n_rails/tasks`, so it's not a problem.